lib/screens/home_screen.dart

```diff
@@ -0,0 +1,196 @@
+import 'package:flutter/material.dart';
+import 'package:furniture/models/items_list.dart';
+import 'package:furniture/models/grid_items_list.dart';
+import 'package:furniture/screens/products_screen.dart';
+
+class HomeScreen extends StatefulWidget {
+  const HomeScreen({Key? key}) : super(key: key);
+
+  @override
+  State<HomeScreen> createState() => _HomeScreenState();
+}
+
+class _HomeScreenState extends State<HomeScreen> {
+  @override
+  Widget build(BuildContext context) {
+    Size size = MediaQuery.of(context).size;
```

Comment:
Dart 3 introduced new way to use inherited widgets to avoid unnecessary rebuilds

MediaQuery.sizeOf(context)